spec/apollo-server-spec.yaml

```diff
@@ -0,0 +1,56 @@
+swagger: "2.0"
+info:
+  description: "This is the backend component API. Visit the link for an interactive playground to execute requests."
```

Comment:
A bit of more detail would be nice, or a high level description of the endpoint. Also mention that you use GraphQL